src/Clerk.ts

```diff
@@ -150,25 +152,29 @@ export default class Clerk {
 
   // Utilities
 
-  decodeToken(token: string): jwt.JwtPayload {
+  decodeToken(token: string): JwtPayload {
     const decoded = jwt.decode(token)
     if (!decoded) {
       throw new Error(`Failed to decode token: ${token}`)
     }
 
-    return decoded as jwt.JwtPayload
+    return decoded as JwtPayload
   }
 
-  async verifyToken(token: string, algorithms: string[] = ['RS256']): Promise<jwt.JwtPayload> {
+  async verifyToken(token: string, algorithms: string[] = ['RS256']): Promise<JwtPayload> {
     const decoded = jwt.decode(token, { complete: true })
     if (!decoded) {
       throw new Error(`Failed to verify token: ${token}`)
     }
 
     const key = await this._jwksClient.getSigningKey(decoded.header.kid)
-    const verified = jwt.verify(token, key.getPublicKey(), { algorithms: algorithms as jwt.Algorithm[], audience: 'clerk'})
+    const verified = jwt.verify(token, key.getPublicKey(), {algorithms: algorithms as jwt.Algorithm[]}) as JwtPayload
 
-    return verified as jwt.JwtPayload
+    if (!verified.hasOwnProperty('iss') || !(verified.iss?.lastIndexOf('clerk.', 0) === 0)) {
```

Comment:
🔧  This starts with `https://clerk.` - see https://github.com/clerkinc/clerk-sdk-ruby/pull/9/files#diff-b8dd9279fc79620beffbf4ca787c70a26d5fe0208f082a64fee4f16211c7340fR178

Reply:
Fixed